### DISCUSSION TITLE
SVG Logo

### DISCUSSION THREAD

**Discussion by tpansino** [cmintey/wishlist#123] [Ideas]
Is the `src/lib/assets/logo.png` file available as an SVG anywhere? If not, can it be made available? It's useful for creating bookmark cards in personal dashboards like [Homer](https://github.com/bastienwirtz/homer) or [Heimdall](https://heimdall.site/).

**Comment by cmintey** [comment]
Unfortunately, no. I drew this logo in [Krita](https://krita.org/en/) which is a rasterized drawing program and so it is unable to export vector graphics like SVG. The png in the assets folder you mentioned is quite high-res and should work well for your uses. I can provide a `.webp` version of the file if that would be better for your use case.

**Reply by tpansino** [reply]
Ah, no worries, I'm content with just the PNG then. Thanks!
